tests/func/test_import.py

```diff
@@ -286,7 +286,7 @@ def test_import_non_existing(erepo_dir, tmp_dir, dvc):
 
     # https://github.com/iterative/dvc/pull/2837#discussion_r352123053
     with pytest.raises(PathMissingError):
-        tmp_dir.dvc.imp(fspath(erepo_dir), "/root/", "root")
```

Comment:
Not sure how this has worked before, since `/root` does exist on linux'es. Might be missing something...